docs/api/menu-item.md

```diff
@@ -48,6 +49,8 @@ See [`Menu`](menu.md) for examples.
     the placement of their containing group after the containing group of the item
     with the specified label.
 
+**Note:** `acceleratorWorksWhenHidden` is specified as being macOS-only becuase the behavior it enables when `true` is the default on Windows and Linux.
```

Comment:
Tyop: becuase

More importantly I don't understand the explanation. If it's useful for app devs to be able to choose to enable or disable this, the platform default doesn't matter?

Or, if the description is saying this is always useful on Win and Linux... would it always be useful on Mac too -- and should always be 'on' and not need a user-visible Mac setting?

Reply:
This is something that is _always_ on for windows and linux; and on Mac it's something that, in native development, is always a choice by the developer. Thus, we now default it to true, to match behavior on other platforms, but allow developers to turn it off for Mac if they wish, since they would be able to do this when developing native Mac apps

(i'll clean up the property description, bc as I typed this i realized my original was not at all clear 😆)

Reply:
So there is no native mechanism for making this choice on Windows?

I'm not understanding why we want to let app devs choose the setting value on Platform A but not let them choose on Platforms B and C.

Reply:
@ckerr primarily, to most closely match what they could do if they were developing a native application for macOS.